ddtrace/tracer/post_processor.go

```diff
@@ -0,0 +1,111 @@
+package tracer
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+// Note: Keeping everything in this file for easy access during development.
+// I will likely move the ReadWriteSpan interface to the ddtrace package,
+// and the (s *span) *() methods to span.go.
+
+// ReadOnlySpan specifies methods to read from a span.
+type readOnlySpan interface {
```

Comment:
I think `readableSpan` sounds nicer IMHO
```suggestion
type readableSpan interface {
```